eman2/convert/convert.py

```diff
@@ -152,6 +170,79 @@ def readCoordinates(mic, fileName, coordsSet, invertY=False):
                 coord.setMicrograph(mic)
                 coordsSet.append(coord)
 
+def readCoordinate3D(box, inputTomo):
+    from pwem import Domain
```

Comment:
This will go away. A plain import would do.

Reply:
Yep, see https://github.com/scipion-em/scipion-em-eman2/commit/3b805869019181e3e364f9837a73ef99f09c12dd